spec/index.md

```diff
@@ -2343,30 +2343,28 @@ The [standard nearest-rank estimation method](https://en.wikipedia.org/wiki/Perc
 ### IC method `fetch_logs` {#ic-fetch_logs}
 
 Given a canister ID as input provides the logs of that canister including trap messages.
-Logs are stored in a circular buffer of 4KiB.
+The total length of all log contents does not exceed 4KiB.
+If new logs are added that exceed the memory capacity, the oldest logs will be removed.
 Log memory persists across upgrades and it is deleted if the canister is reinstalled or uninstalled.
 The log visibility is defined in `log_visibility` field of `canister_settings`.
-Users cannot call this method.
 
 A single log entry is a record with the following fields:
 
 - `idx` (`nat64`): the unique incremental index of an entry;
 - `timestamp_nanos` (`nat64`): the timestamp as nanoseconds since 1970-01-01 at which the entry was recorded;
 - `contents` (`blob`): the content of a log entry;
 
-### IC method `fetch_logs_query` {#ic-fetch_logs_query}
-
-This method is identical to [`fetch_logs`](#ic-fetch_logs), but exposed as a query.
-
 :::note
 
-This query is only accessible in non-replicated mode. Calls in replicated mode are rejected.
+This method is exposed as a query and is only accessible in non-replicated mode. 
+Calls in replicated mode are rejected.
```

Comment:
If this is (supposed to be) equivalent to "Users cannot call this method.", could you please also update the other occurrence of this phrase?

Reply:
in the canister_info section

Reply:
1) `Calls in replicated mode are rejected` is not an equivalent to `Users cannot call this method`, because we want users to call this method, if fact we want ONLY users to call this method (not canisters). and since canisters can make only update calls to other canisters we want `calls in replicated mode are rejected`.

2) I don't feel comfortable modifying specs of other methods in this PR, just not to mess around with the PR scope.

Reply:
> Calls in replicated mode are rejected is not an equivalent to Users cannot call this method

My bad, I'm sorry!